### ISSUE TITLE
Gauge/charts doesn't update when msg.enabled = true

### ISSUE THREAD

**Issue by natcl** [node-red/node-red-dashboard#105]
When sending data and msg.enabled == true the gauge doesn't seem to update, here's an example flow:

```
[{"id":"b02cdd01.e300b","type":"ui_gauge","z":"81ce221c.95b848","name":"","group":"7eb25788.6078d","order":0,"width":0,"height":0,"gtype":"gage","title":"Gauge","label":"units","format":"{{value}}","min":0,"max":10,"colors":["#00B500","#E6E600","#CA3838"],"x":580,"y":280,"wires":[]},{"id":"d8d9c9e8.2282f8","type":"inject","z":"81ce221c.95b848","name":"","topic":"","payload":"40","payloadType":"num","repeat":"","crontab":"","once":false,"x":190,"y":280,"wires":[["b02cdd01.e300b","bb958bdb.955878"]]},{"id":"a6b1fa66.bce068","type":"ui_gauge","z":"81ce221c.95b848","name":"","group":"7eb25788.6078d","order":0,"width":0,"height":0,"gtype":"gage","title":"Gauge","label":"units","format":"{{value}}","min":0,"max":10,"colors":["#00B500","#E6E600","#CA3838"],"x":590,"y":340,"wires":[]},{"id":"bb958bdb.955878","type":"change","z":"81ce221c.95b848","name":"","rules":[{"t":"set","p":"enabled","pt":"msg","to":"true","tot":"bool"}],"action":"","property":"","from":"","to":"","reg":false,"x":350,"y":320,"wires":[["a6b1fa66.bce068"]]},{"id":"510b5e44.7d2418","type":"inject","z":"81ce221c.95b848","name":"","topic":"","payload":"0","payloadType":"num","repeat":"","crontab":"","once":false,"x":190,"y":140,"wires":[["b02cdd01.e300b","a6b1fa66.bce068"]]},{"id":"c17b1f1d.6f5c08","type":"comment","z":"81ce221c.95b848","name":"Fails for second gauge","info":"","x":220,"y":380,"wires":[]},{"id":"7eb25788.6078d","type":"ui_group","z":"","name":"Third","tab":"df04737a.8811d8","order":6,"disp":true,"width":"6"},{"id":"df04737a.8811d8","type":"ui_tab","z":"","name":"Software monitoring","icon":"dashboard"}]
```

**Comment by dceejay** [comment]
closed by #106

**Comment by natcl** [comment]
Thanks works like charm !
